core/src/main/java/org/stellar/anchor/sep12/Sep12Service.java

```diff
@@ -56,8 +67,19 @@ public Sep12PutCustomerResponse putCustomer(Sep10Jwt token, Sep12PutCustomerRequ
       request.setAccount(token.getAccount());
     }
 
-    return PutCustomerResponse.to(
-        customerIntegration.putCustomer(PutCustomerRequest.from(request)));
+    PutCustomerResponse response =
+        customerIntegration.putCustomer(PutCustomerRequest.from(request));
+
+    // Only publish event if the customer was updated.
+    eventSession.publish(
+        AnchorEvent.builder()
+            .id(UUID.randomUUID().toString())
+            .sep("12")
+            .type(AnchorEvent.Type.CUSTOMER_UPDATED)
+            .customer(CustomerUpdatedResponse.builder().account(request.getAccount()).build())
```

Comment:
I don't think this is the right interface for the event. We want to move away from using Stellar accounts to identify the customer. Can we use the customer ID returned in the response instead?